### PR TITLE
V4

### PR DESCRIPTION
**Pull request by arao6**
SDK V4 upgrades into master.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <packaging>jar</packaging>
 
     <name>hyperwallet-java-sdk</name>
-    <description>An SDK implementation in Java for the v3 REST APIs.</description>
+    <description>An SDK implementation in Java for the v4 REST APIs.</description>
```
Comment (peter-joseph) [inline pom.xml:11]: noticed that version is still  `<version>1.4.3-SNAPSHOT</version>` I thought we reserved `1.x.x` as V3 and `2.x.x` as V4 ? or any equivalent?

File: pom.xml [hunk 2]
```diff
@@ -255,7 +268,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-failsafe-plugin</artifactId>
-                <version>2.20.1</version>
+                <version>2.22.2</version>
                 <executions>
                     <execution>
                         <configuration>
```
Comment (peter-joseph) [inline pom.xml:274]: thanks for updating to new versions 👍

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 3]
```diff
@@ -20,7 +21,6 @@
 public class Hyperwallet {
 
     public static final String VERSION = "1.4.2";
```
Comment (peter-joseph) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:23]: noticed that version is still  `VERSION = "1.4.2"` I thought we reserved `1.x.x` as V3 and `2.x.x` as V4 ? or any equivalent?

**Review by peter-joseph — CHANGES_REQUESTED** [review]
I'm also expecting that MD file will be updated, based on option 3 design as agreed, since master is now V4, and there will be a link to branch V3 and then in branch V3 the MD file for that branch will also mention that it will only support V3 REST api?

File: pom.xml (hunk 1 above)
Comment (arao6) [reply pom.xml:11]: That's right, I updated Hyperwallet.java, README.md and CHANGELOG.md with 2.0.0 as suggested by Willian. 
He said it was ok to let the version stay the same in pom.xml for now. 
I believe the version will be updated by the integrator.

Comment (arao6) [reply pom.xml:11]: I pushed the changes once again, looks like they didnt make it last time.

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java (hunk 3 above)
Comment (arao6) [reply src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java]: Willian suggested that it should be 2.0.0 
I've updated the files with that (except pom.xml)

**Comment by arao6** [comment]
Merge conflicts have been resolved

**Review by peter-joseph — APPROVED** [review]
LGTMN 👍

**Pull request merged**